allocator/src/lib.rs

```diff
@@ -173,6 +150,18 @@ impl GcAllocator {
     pub fn init() {
         unsafe { boehm::GC_start_performance_measurement() };
     }
+
+    pub unsafe fn register_thread(stack_base: *mut u8) -> bool {
```

Comment:
What do the `bool` return types mean? It might be worth briefly documenting them.

Reply:
Sure, they indicate whether the thread was successfully (un)/registered. I'll document this.

Reply:
Fixed in f804f9f